libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/voice/AndroidSpeechPlayer.java

```diff
@@ -46,6 +49,11 @@ public void onInit(int status) {
   @Override
   public void play(String instruction) {
     if (!isMuted && !TextUtils.isEmpty(instruction)) {
+
+      if (Build.VERSION.SDK_INT < Build.VERSION_CODES.ICE_CREAM_SANDWICH_MR1) {
```

Comment:
Can we extract this into a private method with naming that explains we are firing the listener only for `ICE_CREAM_SANDWICH_MR1`?